### PR TITLE
Create sickness_remover.py

### PR DESCRIPTION
**Pull request by CompuGenius**
This file will fix all sickness that plagues the Earth.

### CODE REVIEW

**Comment by CompuGenius** [comment]
> Simplify imports otherwise performance will effect the tide, sunrise etc.

How would you suggest I go about that? I'm new the complexities of earth.

File: sickness_remover.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+for human in range(len(earth.humans)):
+    if human.is_sick:
+      human.is_sick = False
+    if not human.has_antibodies:
+      human.has_antibodies = True
+      
+earth.humans.need_to_mask = False
+earth.humans.need_to_quarantine = False
+
+for sickness in earth.sicknesses:
```
Comment (gerslun) [inline sickness_remover.py:13]:
```suggestion
humans.need_to_mask = False
humans.need_to_quarantine = False

for sickness in sicknesses:
```

File: sickness_remover.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from earth import humans
+from earth import sicknesses
+
+for human in range(len(earth.humans)):
```
Comment (gerslun) [inline sickness_remover.py:4]: Same with this. You might even kill some humans.

**Comment by CompuGenius** [comment]
> redundant if statements
I have fixed. I was unsure if not checking would cause unnecessary calls, slowing down Earth, thereby causing problems with the day/night cycle, but all seems to be working.

**Pull request merged**